e2e/db.js

```diff
@@ -0,0 +1,39 @@
+import mongoose, { mongo } from "mongoose";
+import { MongoMemoryServer } from "mongodb-memory-server";
+
+let mongoServer = null;
+let dbInstance = undefined;
+let MONGO_PORT = process.env.MONGODB_PORT || 10000;
+let MONGO_URL = `mongodb://127.0.0.1:${MONGO_PORT}/`;
+
+const connectDB = async () => {
+  try {
+    await mongoose.disconnect();
+    mongoServer = await MongoMemoryServer.create({
+      instance: {
+        port: +MONGO_PORT,
+      },
+    });
+    dbInstance = await mongoose.connect(MONGO_URL);
+  } catch (error) {
+    console.log("Mongo db connect error: ", error);
```

Comment:
Take a pull from `main`, we have configured a `winston` logger. use `logger.error()` wherever you are logging an error. Also wherever you need to log something use `logger` module only. Don't use `console` anywhere.

Reply:
Agree, `feat/coverage` is 18 commits behind `main` so would be a good idea to take latest changes

Reply:
as per my opinion using logger in testing is not right way logger is used to log application logs not testing logs 
and playwrite create report file where it show logs as well 

egarding merging `master` to  `feat/coverage`  when we are planing to do that after this PR merged?  because there are conflict that we have to resolve  
@wajeshubham

Reply:
@shrey-dadhaniya I understand, but using `logger` instead of `console.log` will result in comprehensive logs in the playwright report as well as keeps code consistent and more readable. I believe it does not affect how playwright deals with logs in the test files.

Reply:
Sounds good! be careful while resolving the merge conflicts.